pkg/collector/corechecks/ebpf/tcpqueuelength/tcp_queue_length.go

```diff
@@ -110,9 +110,9 @@ func (t *TCPQueueLengthCheck) Run() error {
 			continue
 		}
 
-		entityID := types.NewEntityID(types.ContainerID, containerID).String()
+		entityID := types.NewEntityID(types.ContainerID, containerID)
 		var tags []string
-		if entityID != "" {
+		if entityID.Empty() {
```

Comment:
This change looks counter-intuitive to me because I would have expected `""` to be equivalent to `.Empty()` and not the other way round.

Shouldn’t this be:
```suggestion
		if entityID.NotEmpty() {
```
or
```suggestion
		if !entityID.Empty() {
```
?